hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/StorageContainerManagerStarter.java

```diff
@@ -112,6 +112,25 @@ public void initScm(@CommandLine.Option(names = { "--clusterid" },
     }
   }
 
+  /**
+   * This function implements a sub-command to allow the SCM to be
+   * initialized from the command line.
+   */
+  @CommandLine.Command(name = "--bootstrap",
+      customSynopsis = "ozone scm [global options] --bootstrap",
```

Comment:
Question: [global options] what are these global options?

Reply:
[global options]  are hadoop generic options . I guess, maintained for legacy same as in scm --init.